bylaws/001-mission-and-structure.md

```diff
@@ -24,59 +24,63 @@ A Member Project may not have more than one Project Representative at a time. No
+The Core Committee is a twelve (12) member board of individuals recognized for their technical skill and expertise. The Core Committee is responsible for final decisions regarding what specifications FIG will consider and those that are approved. The Core Committee is responsible for ensuring a high level of quality and consistency amongst all published specifications, and that all relevant perspectives and use cases are given due consideration.
 
 Core Committee members are not permitted to also be a Secretary. A Project Representative is permitted but not required to also serve as a member of the Core Committee, however, Core Committee members are expected to consider the impact of their actions on the PHP ecosystem as a whole when acting in their capacity as a Core Committee member
 
 Core Committee members are expected to take an active interest in all topics brought before FIG.
 
 ## Secretaries
 
-The primary responsibility of the FIG Secretary is to serve as an impartial administrator of the FIG. They serve at the member projects pleasure but act as independent and impartial adjudicators. FIG Secretaries will also represent the FIG as a whole to the general community and public on matters relating to the FIG's activities.
+The primary responsibility of the FIG Secretary is to serve as an impartial administrator of the FIG. They serve at the member projects and core committee's pleasure but act as independent and impartial adjudicators. FIG Secretaries also represent the FIG as a whole to the general community and public on matters relating to the FIG's activities.
 
 Whilst there are a number of defined functions below but they should also perform other duties as required. As the role has continuity (in that there will always be a post holder) and redundancy (There are a number of post holders in case of the absence of one) it can be ensured that responsibilities assigned to the secretary will always be completed and therefore administrative responsibilities such as vote management should always be assigned to the FIG Secretary.
 
 All FIG Secretaries are expected to remain impartial and professional when acting in an official capacity as Secretary and should also remain aware that even when not acting in an official capacity, their actions reflect back on the PHP FIG and the FIG must not be brought into disrepute.
 
-In order to ensure impartiality, Secretaries may not concurrently serve as a Project Representative, Core Committee Member, or Editor of a PSR. They may, however, be a member of a Working Group.
+In order to ensure impartiality, Secretaries may not concurrently serve as a Project Representative, Core Committee Member, or Editor of a PSR. They may, however, be a member of a Working Group. They must declare all conflicts of interest (for example being on the core team of member projects or involvement in PSR working groups)
```

Comment:
Probably needs an "in that case" or something?  They must declare such where, when, how?